src/test/java/com/sproutsocial/nsq/PublisherWithFailoverDockerTestIT.java

```diff
@@ -59,6 +60,32 @@ public void withBackup_failoverAndFailBackProactivly() {
         sendAndVerifyMessagesFromPrimary(publisher, handler);
     }
 
+    @Test
+    public void clearsConnectionsIfAllNodesAreMarkedDead() {
+        // Force publishing to fail by disconnecting all the nodes network
+        // Perhaps to simulate a bad network flake.
+        cluster.disconnectNetworkFor(cluster.getNsqdNodes().get(0));
```

Comment:
Nit: alternative?  No strong preference.  
```suggestion
        cluster.getNsqdNodes().forEach(cluster::disconnectNetworkFor); 
```

Reply:
You can tell I was tired when I wrote this code last night. 😆 Will update.